image.yaml

```diff
@@ -103,8 +103,8 @@ modules:
           - name: sso.config.launch.setup.74
           - name: sso.db.drivers
             version: '1.0'
-          - name: sso.datasource.overrides
-            version: '1.0'
+          # - name: sso.datasource.overrides
+          #   version: '1.0'
```

Comment:
@hmlnarik Do we want to remove this module altogether or intentionally keeping it for any future re-use?

Reply:
Thanks @iankko. The overrides were there to account for mariaDB, so this module might not be necessary anymore. I need to check whether the without the overrides, the image behaves correctly. If so, we can remove it completely.